src/Symfony/Bridge/PhpUnit/ErrorAssert.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\Bridge\PhpUnit;
+
+/**
+ * Test that your code triggers expected error messages.
+ *
+ * @author Christian Flothmann <christian.flothmann@xabbuh.de>
+ */
+final class ErrorAssert
+{
+    public static function assertDeprecationsAreTriggered($expectedMessages, callable $testCode)
```

Comment:
Given that the method takes an array of expected messages, plural is fine.

``setExpectedDeprecation`` is not fine for the current API. It would work if the method was register the expectation to check it later after running the test, but this is not what happens in the current API.